src/gsi/convthin.f90

```diff
@@ -464,7 +464,7 @@ subroutine map3grids_m(flg,pflag,pcoord,nlevp,dlat_earth,dlon_earth,pob,crit1,io
 
     real(r_kind) dlat1,dlon1,pob1
     real(r_kind) dx,dy,dp
-    real(r_kind) dxx,dyy,dpp
+!   real(r_kind) dxx,dyy,dpp
```

Comment:
If this line is not needed, we should remove it.

Reply:
Removed.